pnpm-workspace.yaml

```diff
@@ -2,8 +2,8 @@ packages:
   - packages/*
 catalog:
   # deltachat specific
-  '@deltachat/jsonrpc-client': v1.151.1
-  '@deltachat/stdio-rpc-server': v1.151.1
+  '@deltachat/jsonrpc-client': v1.151.2
+  '@deltachat/stdio-rpc-server': v1.151.2
   webxdc-types: ^1.0.1
```

Comment:
might make sense to also update those webxdc types. the package was renamed to `@webxdc/types`

https://github.com/webxdc/webxdc-types

Reply:
So because of recent type confusion, I now believe this should be done in this pr.